apps/system/lockscreen/js/lockscreen_inputpad.js

```diff
@@ -117,11 +115,11 @@
         if (!key) {
           break;
         }
-
         // Cancel the default action of <a>
         evt.preventDefault();
         this.handlePassCodeInput(key);
-      break;
+        // .handlePassCodeInput() triggers .updatePassCode()
```

Comment:
It's not code, just a comment, and I think it's necessary to follow the logic without looking any further.